definitions/defaults/virtual_keyboard.lua

```diff
@@ -0,0 +1,36 @@
+-- todo
+--
+--  i believe that I should be able to test now?!
+--
+--  it is quite annoying that i feel so annoyed by sitng at the cmputer.
+--  but i really feel now that thingr are gong to get really nice.
+--
+--  i do believe that it hould be possible to do this machine learning hing
```

Comment:
there's a lot of comments that shouldn't go here, but I see you're typing something for me. Machine learning in reaper sounds interesting actually! what did you have in mind? I'm really interested in the intersection of code and music as well. Recently I got into the VCV Rack environment and programmed some modules there, maybe you will like that program too. It's philosophy is very Unix like 'do one thing and do it well'.

Reply:
Cool! Sometimes I kind of 'write my mind' in comments so I don't forget stuff. In my case I am a beginner at machine learning but I have friends who do it and I have taken some courses on MIR stuff. In engineering school I used some supercollider and in the future I have been thinking it would be cool to use reaper in conjunction with supercollider and apply some ML on top of it and connect it to some web app maybe. I don't know really but I think that the best way for learning is to apply it on what I love. Let's make stuff as cyber as possible! My goal is basically to just learn as much about how to handle stuff in code and commndline now. I am starting to get good at computers now and so I just want to do stuff to learn as much as possible.
I will check out VCV. I recall the name!!

Reply:
Also i’ve had repetitive strain stuff and so is sometimes just write random stuff to diagnose myself and that’s why you might see some incomprehensible comments  😁